src/provider/view/components/NotificationView/NotificationView.tsx

```diff
@@ -60,19 +60,22 @@ export class NotificationView extends React.Component<NotificationViewProps, Not
     public componentDidMount(): void {
         fin.desktop.main(async () => {
             const allNotifications = await window.openfin.notifications.fetchAllNotifications();
+            allNotifications.forEach(notif => {
+                notif.notification.date = new Date(notif.notification.date);
```

Comment:
What's happening here? Are we expecting date to come back as a number? And isn't this already happening inside fetchAllNotifications

Reply:
The date that we get here has been sent over the bus, and so been serialised. It's not that it's a number per-se, just that it's missing the functions that we need in other place. 

Also, this was just copying what was already being done in other places in the centre UI, and is likely to change soon when the multi-window redux stuff comes in.